sky/serve/autoscalers.py

```diff
@@ -90,7 +103,7 @@ class RequestRateAutoscaler(Autoscaler):
     """
 
     def __init__(self, spec: 'service_spec.SkyServiceSpec',
-                 qps_window_size: int) -> None:
+                 qps_window_size: int, initial_version: int) -> None:
```

Comment:
Just curious, why we need to pass int a initial_version in the argument, instead of directly set it to the constant during the initialization?

Reply:
In the future there might be a case where the user wants to instantiate from a specific version;
but currently it is set to the constant.

Reply:
removed